test/system/src/main/java/io/pravega/test/system/framework/services/docker/PravegaControllerDockerService.java

```diff
@@ -66,7 +66,7 @@ public void start(final boolean wait) {
     }
 
     private ServiceSpec setServiceSpec() {
-        Mount mount = Mount.builder().type("Volume").source("volume-logs").target("/tmp/logs").build();
+        Mount mount = Mount.builder().type("Volume").source("volume-logs").target("/opt/pravega/logs").build();
```

Comment:
@RaulGracia Does this implies the cluster node has  a volume `/volume-logs` for controller instances and `/logs-volume` for all segmentstore instances?

Also, if a segmentstore is stopped and scheduled again on the same slave node will it cause the logs files to be overwritten?

Reply:
The volume does not necessary need to exist in the cluster node. If it doesn't exist, it will be created dynamically, otherwise, it'll be reused.

However, the volume names used (`volume-logs`, `logs-volume`) do not look good. Maybe we could change that to `logs-controller` and `logs-segmentstore`, or similar.